app/src/main/java/com/nitrr/ecell/esummit/ecellapp/activities/Login.java

```diff
@@ -24,11 +29,22 @@
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_login);
-
     }
 
     public void onGClick(View view) {
         // Build a GoogleSignInClient with the options specified by gso.
         GoogleSignInClient mGoogleSignInClient = GoogleSignIn.getClient(this, gso);
     }
+
+    public void ontransiton(View view) {
+        LoginAnimation loginanimation = new LoginAnimation(this);
+        loginanimation.doanimation(this);
+        Utils u=new Utils();
```

Comment:
Name.
Make functions of utils static. No need to create multiple instance of that class.

Reply:
Done!